<launch>
    <node pkg="hrl_clickable_world" name="clickable_display" type="clickable_display">
        <remap from="image" to="/clickable_world/image_buttons" />
    </node>
</launch>
